src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -218,31 +218,75 @@ object AddDatasetMetadataToDeposit {
   }
 
   def createSpatialPoints(dataset: Dataset): Seq[Elem] = {
-    // coordinate order latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
-    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues =>
+    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues => {
+      val srsName = createSrsName(mdKeyValues)
+      // coordinate order x, y = longitude (DCX_SPATIAL_X), latitude (DCX_SPATIAL_Y)
+      val xy = s"${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")}"
+      // coordinate order y, x = latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
+      val yx = s"${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")}"
+
+      val pos = srsName match {
+        case "http://www.opengis.net/def/crs/EPSG/0/28992" => xy
+        case "http://www.opengis.net/def/crs/EPSG/0/4326" => yx
```

Comment:
both this value and ALL OTHER values return `yx`. Why not merge them by deleting this line? I guess you have a special reason for that?